hopsworks-web/yo/app/views/featurestore.html

```diff
@@ -610,5 +619,79 @@
                 </div>
             </div>
         </div>
+        <tour class="ng-scope" step="featurestoreCtrl.tourService.currentStep_TourNine"
+              post-tour="closeTour()"
+              ng-if="featurestoreCtrl.tourService.activeTour === 'featurestore'
+          && (featurestoreCtrl.tourService.informAndTips
+              || featurestoreCtrl.tourService.tipsOnly) && !featurestoreCtrl.loading && !projectCtrl.loading">
+            <virtual-step
+                    ng-if="featurestoreCtrl.tourService.activeTour === 'featurestore'"
+                    class="ng-scope"
+                    tourtip="This is the feature store registry where you can list and search for features/feature groups
+                    and training datasets in the feature store.
+                    You can also edit metadata of features/feature groups and training datasets from here"
```

Comment:
@Limmen we could reduce the text, like `This is the feature store registry where you can list and search for features/feature groups training datasets in the feature store as well edit their metadata.` ?

Reply:
Done